<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_SetGridType</class>
 <widget class="QWidget" name="QG_SetGridType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>37</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>6</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbGap">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>7</y>
     <width>38</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>间隔</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setGridGap">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>7</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbInfo">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网格参数:  类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_setGridType">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>7</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string> 虚线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 实线</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slotNext()</slot>
  <slot>slotOk()</slot>
 </slots>
</ui>
